### DISCUSSION TITLE
Moderator for Github Discussions

### DISCUSSION THREAD

**Discussion by vinckr** [community/community#3029] [Discussions] [answered]
Afaik as of now it is only possible to moderate a discussions board when you are owner, triager or maintainer of a repository. 

It would be awesome if there were some way that we could designate certain users as "discussion moderators" giving them the permissions to: 
- select an reply as answer "mark as answer"
- close threads
- delete or flag inappropriate replies 
(- possibly move discussions to issues and vice versa)

Especially the "mark an answer" permission would be useful. 

Is there some other way to solve this, that I have not figured out? Let me know!

**Comment by evi-liu** [comment, marked as answer]
As you mentioned, those with triage+ permissions currently have access to those capabilities. I'm curious why you'd suggest a separate `discussion moderator` permission when you can just give those users `triage` permissions. Curious to hear thoughts!

**Reply by vinckr** [reply]
Thanks for your reply! 
You are right, I was looking for the capabilities that the `triage` permission grants!

The only difference from the role I imagined is, that it also gives you permission for closing issues ect., but then again it makes sense to be able to convert issues to discussions and back. 

Is there some portion of the github documentation where I could add this information, or have I just overlooked it?

**Reply by evi-liu** [reply]
I think the [documentation here](https://docs.github.com/en/github/setting-up-and-managing-organizations-and-teams/repository-permission-levels-for-an-organization) provides you with the information you're looking for!
